articles/openstack/ostack-ref-public-image-versioning.md

```diff
@@ -19,30 +19,24 @@ toc_mdlink: ostack-ref-public-image-versioning.md
 
 When you create an instance in OpenStack, its disk is cloned from an image managed by OpenStack. To get you started quickly, we publish a public catalog on our platform that provides several instance images for commonly-used instance sizes and operating systems.
 
-To provide you with both predictability and access to the latest updates, without having to provide an unmanageably large catalog, we use a long-term support (LTS) approach for our OpenStack public image catalog.
-
 ## Image versioning
 
-In our public catalog, we keep the original version of the image in the catalog for the lifetime of that distribution (providing predictability). We also maintain a second version of the release of the template that we patch and replace as necessary during the distribution (providing you with access to the latest version).
-
-This means that:
-
-- The LTS (original version) is predictable both in deployment state and UUID within OpenStack
+We will be providing updated images once a quarter for our supported images. We will not be removing images from the platform until they are no longer in use by any customers. We will be adopting a naming convention that will allow you to easily identify images.
```

Comment:
I'd suggest changing the tense of this to be a bit simpler:

We update our supported images once a quarter. We will not remove images from the platform until they are no longer in use by any customer.

I'd also suggest moving the bit about naming conventions to the next section.